### ISSUE TITLE
Option to disable "buy ad-free version" message

### ISSUE THREAD

**Issue by AlienCreature7** [sadiqrazasyed/wallpaperboard-enhanced#8]
I want to disable this message in the app because the app I'm working on right now doesn't have an ad-free version. I can't find how to disable it.

**Comment by genzcodersz** [comment]
Replace **WallpaperBoardActivity.java** & **WallpaperBoardPreviewActivity.java** with these -  [LINK](https://drive.google.com/open?id=1VlMKVr8zjdSieiOkpddjdrK5Yq6Md8v8)

**Comment by AlienCreature7** [comment]
Those files are not accessible when im using sample project, right?

**Comment by genzcodersz** [comment]
Yep. You need to replace those files in the actual library. Just download/clone the project as it also contains the sample app.

**Comment by AlienCreature7** [comment]
i wanted to avoid using the actual library since it gets messy and sometimes confusing for a person who doesn't do programming. ill try it.

**Comment by genzcodersz** [comment]
Once you import the project in android studio you can see two modules, namely "**app**" & "**library**". Use the app module just like you do with the sample project & you won't have any difficulties.

**Comment by AlienCreature7** [comment]
Thanks. I will do it. 

But it would be good if you also add the option in the sample version.
